### PR TITLE
Adding Bohemian Rhapsody Feb Song

### PR DESCRIPTION
**Pull request by juromerop**
## Describe your changes
The February song for Queen is Bohemian Rhapsody.
## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [x] My code works exceptionally

### CODE REVIEW

File: app/index.md [hunk 1]
```diff
@@ -6,18 +6,6 @@ Welcome to _Onlyrics_, the only place on the planet where your ears won't be dis
...
-[Imagine](song/jan/vanilla-panna-cotta.md)
-
 ### [Queen](writer/queen.md) | queen@onlyrics.magazine
 
-[We Are The Champions](song/jan/we_are_the_champions.md)
-
-### [The Killers](writer/the_killers) | the.killers@onlyrics.magazine
-
-[Mr. Brightside](song/jan/mr_brightside.md)
+[Bohemian Rhapsody](../song/feb/bohemian_rhapsody.md)
```
Comment (escobartc) [inline app/index.md:11]: Good job!

File: app/song/feb/bohemian_rhapsody.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+3. The song was released as a single on 31 October 1975, with “I’m in Love with My Car” as the B-side1. It was initially met with skepticism by the record company and some critics, who thought it was too long and complex for radio play. However, it became a huge success after being championed by DJ Kenny Everett, who played it repeatedly on his show. It topped the UK Singles Chart for nine weeks and became the UK’s third best-selling single of all time. It also reached number one in several other countries and number nine in the US.
+
+4. The song was accompanied by a promotional video, which is considered one of the first and most influential music videos ever made. It was directed by Bruce Gowers and featured the band miming to the song in a darkened studio, using special effects and lighting. The video was based on a photo of the band by Mick Rock, inspired by Marlene Dietrich2. The video helped popularise the song and the medium of music video, as it was shown on various TV shows around the world.
```
Comment (escobartc) [inline app/song/feb/bohemian_rhapsody.md:19]: Changes looks good

**Review by escobartc — APPROVED** [review]
Looks pretty good, approved

**Review by josanchezva — APPROVED** [review]
Looks good! I think it is ready to merge.

**Pull request merged**